extensions/amp-access/0.1/login-dialog.js

```diff
@@ -267,7 +267,7 @@ export class WebLoginDialog {
     }
     dev().fine(TAG, 'Login done: ', result, opt_error);
     if (opt_error) {
-      this.reject_(opt_error);
+      this.reject_(user().createError('login failed', opt_error));
```

Comment:
From what I can find, the only places this error can be are:

- https://github.com/jridgewell/amphtml/blob/ccbe994f5372b65173e1d2d224c74ef3cc6a07a6/extensions/amp-access/0.1/login-dialog.js#L203
- https://github.com/jridgewell/amphtml/blob/ccbe994f5372b65173e1d2d224c74ef3cc6a07a6/extensions/amp-access/0.1/login-dialog.js#L215

It could also be from a [rejected `#urlOrPromise`](https://github.com/jridgewell/amphtml/blob/ccbe994f5372b65173e1d2d224c74ef3cc6a07a6/extensions/amp-access/0.1/login-dialog.js#L198), which can never happen. So it's always an unmarked Error instance. I could change this to `dev().assertError()`, but then we'd lose the `"login failed"` context in the error message.